js/exercise-03.js

```diff
@@ -0,0 +1,23 @@
+/* 
+    1. Define function 'getTypeOf', which receives a value as paramenter and returns the type of this value
+        Use typeof to ckech the types for each of the main 5 primitive types (string, number, boolean, null, undefined)
+    2. Log the result of any function call.
+*/
+
+//Declare function
+function getTypeOf(value) {
+    return typeof value;
+}
+
+//Output function
+console.log(getTypeOf("Hello cats!")); 
```

Comment:
Which is the output of these type evaluations?

```suggestion
console.log(getTypeOf("Hello cats!"));  // string
```

FIX: add logs output